### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
In this PR, I have completed the following tasks.

Created a database named vet_clinic.
Created a table named animals.
Inserted data for 4 animals.
Wrote queries according to the requirements.
Took a screenshot of the results of the queries.
SC-1
SC-2

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @Talha-Hanif5666 👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
You have submitted a project that doesn't have All main headers from the [README template](https://github.com/microverseinc/readme-template) 😨, Kindly fix this and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @Talha-Hanif5666 👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
You have submitted a project that doesn't have All main headers from the [README template](https://github.com/microverseinc/readme-template) 😨, Kindky add the **License, 🔭 Future Features, Acknowledgments** and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

**Review by CollinsTatang — COMMENTED** [review]
Hi @Talha-Hanif5666, 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an updated README file.

- [ ] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [ ] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -14,228 +12,145 @@ This repository includes files with plain SQL that can be used to recreate a dat
...
-- **[key_feature_3]**
+- **Create animal table**
+- **STore data into table**
+- **Query data**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (zainsadaqat) [inline README.md:65]: [Optional]
You may remove the Live Demo section from the README file. As it won't be needed in the Database project 👍

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @Talha-Hanif5666 👋 

## Things to Appreciate 
- [ ] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [ ] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**